bash-preexec.sh

```diff
@@ -261,18 +232,14 @@ __bp_install() {
         shopt -s extdebug > /dev/null 2>&1
     fi;
 
-
-    local existing_prompt_command
-    existing_prompt_command=$(__bp_prompt_command_with_semi_colon)
-
     # Install our hooks in PROMPT_COMMAND to allow our trap to know when we've
     # actually entered something.
-    PROMPT_COMMAND="__bp_precmd_invoke_cmd; ${existing_prompt_command} __bp_interactive_mode"
+    PROMPT_COMMAND="__bp_precmd_invoke_cmd; __bp_interactive_mode"
     eval "$__bp_trap_install_string"
 
     # Add two functions to our arrays for convenience
     # of definition.
-    precmd_functions+=(precmd)
+    precmd_functions+=(precmd, original_prompt_command)
```

Comment:
Can you update the comment as well to reflect adding the original prompt command?